android/assets/jsons/translations/Galician.properties

```diff
@@ -0,0 +1,6481 @@
+Found Religion = Fundar Relixión
+Found Pantheon = Fundar Panteón
+Reform Religion = Reformar Relixión
+Expand Pantheon = Expandír Panteón
+Follow [belief] = Seguir [belief]
+Religions and Beliefs = Relixións e Crenzas
+Majority Religion: [name] = Relixión Maioritaria: [name]
++ [amount] pressure = + [amount] presión
+Holy City of: [religionName] = Ciudad Santa de:[religionName]
+Former Holy City of: [religionName] = Antiga Cidade Santa de: [religionName]
+Followers = Seguidores
+Pressure = Presión
+
+# Religion overview screen
+Religion Name: = Nome de Relixión:
+Pantheon Name: = Nome do Panteón:
+Founding Civ: = Civ. fundadora:
+Holy City: = Ciudad Santa:
+Cities following this religion: = Cidades seguindo esta relixión:
+Followers of this religion: = Seguidores desta relixión:
+Click an icon to see the stats of this religion = Toca unha icona para ver as estatísticas desta relixión
+Religion: Off = Relixión: Desactivada
+Minimal Faith required for\nthe next [Great Prophet]: = Fé mínima requirida para\nel seguinte [Great Prophet]:
+Religions to be founded: [amount] = Relixións a fundar: [amount]
+Available religion symbols = Símbolos relixiosos dispoñibles
+Number of civilizations * [amount] + [amount2] = Número de Civilizacións * [amount] + [amount2]
+Estimated Number of civilizations * [amount] + [amount2] = Número Estimado de Civilizacións * [amount] + [amount2]
+Religions already founded = Relixións xa fundadas
+Available founder beliefs = Crenzas de Fundador dispoñibles
+Available seguidor beliefs = Crenzas de Seguidor dispoñibles
+Religious status: = Estado relixioso:
+
+None = Ningún
+Pantheon = Panteón
+Founding religion = Fundando relixión
+Religion = Relixión
+Enhancing religion = Realzando relixión
+Enhanced religion = Relixión realzada
+
+# Espionage
+# As espionage is WIP, these strings are currently not shown in-game, 
+# so feel free to not translate these strings for now
+
+Spy = Espía
+Spy Hideout = Axencia Espía
+Spy present = Espía Presente
+Move = Mover
+
+After an unknown civilization entered the [eraName], we have recruited [spyName] as a spy! = ¡Despois de que unha civilización descoñecida entrou na [eraName], recrutamos un Espía chamado [spyName]!
+We have recruited [spyName] as a spy! = ¡[spyName] é o novo espía recrutado!
+A spy from [civilization] stole the Technology [techName] from [cityName]! = ¡Un espía de [civilization] roubou a Tecnoloxía [techName] desde [cityName]!
+An unidentified spy stole the Technology [techName] from [cityName]! = ¡Un espía descoñecido roubou a Tecnoloxía [techName] desde [cityName]!
+Your spy [name] stole the Technology [techName] from [cityName]! = ¡O teu espía [name] roubou a Tecnoloxía [techName] desde [cityName]!
+Your spy [name] cannot steal any more techs from [civilization] as we've already researched all the technology they know! = ¡O teu espía [name] non puido roubarlle máis tecnoloxías a [civilization], porque xa descubrimos todas as tecnoloxías que eles teñen!
```

Comment:
Ok, I think that now I understand what the problem was, but I say I think, I'm not sure, I think that the best solution might be for you to tell me where to get the most updated master and do it from that one and not try to repair this one, the problem was that I was I'm sure it wouldn't change the lines to the left of the = and that was what had me crazy.

This would be the teacher I have to start from, correct? I think he was the one I used but I'm not sure about anything right now.

https://github.com/yairm210/Unciv/blob/master/android/assets/jsons/translations/template.properties

The intention would be to do it offline with a text editor or similar and upload it again when I have it repaired, it would take me a long time but given the problems it may be the best solution given the numerous problems.

Reply:
No, I didn't try anything you mentioned for a reason that I explain below, but your help finally made me find the problems, since before I couldn't understand why the translation didn't work, the initial template was not correct and that I could excuse myself, but it is also true that even that one has unforgivable flaws that are 100% solely my fault, just to tell you that after reviewing the first 500 lines I may have repaired close to 100 that were wrong, the best idea I had is to take it again master template in English and do everything again, taking advantage of how much I have already done, now I am going to test whether those 500 lines already translated give errors or not and continue the translation like this if not, it gives me more problems. That translation is more work to correct than to redo it from scratch, I assure you, but it does serve as support for the new one and saves me a lot of work, thank you 1000 for that to you and to the others who helped me see the problem.

P.S: the problem, apart from the markers, is that without being clear about the reason, the original lines also come out with translated words, for example, many "No" come out as "Non" and that had to be me to blame, but I was not aware to do it at any time, and since I only reviewed the markers left and right I would never have found the error.